src/main/java/core/basesyntax/service/impl/operation/service/PurchaserServices.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl.operation.service;
+
+import core.basesyntax.service.OperationService;
+
+public class PurchaserServices implements OperationService {
+    @Override
+    public int newAmount(int oldAmount, int quantityFromTransaction) {
```

Comment:
Usually the name for the main method of strategy class is "handle" or "process". 
In your case it would be cool to have such a naming composition:
Interface - `OperationHandler`
Class - `PurchaseOperationHandler`
Main method - `handle`